js/supermarkets.js

```diff
@@ -3,30 +3,56 @@ let infoWindow;
 let map;
 
 /**Connecting Map Location API  */
-    /*Creating google map*/
-    function initMap() {
-        // The location of Munich
-        const munich = { lat: 48.137154, lng: 11.576124 };
-        // The map, centered at Munich
-        map = new google.maps.Map(document.getElementById("map"), {
-        //  maxZoom: 20,
-            center: munich,
-        });
-        infoWindow = new google.maps.InfoWindow()
-    }
+/*Creating google map*/
+function initMap() {
+    // The location of Munich
+    const munich = { lat: 48.137154, lng: 11.576124 };
+    // The map, centered at Munich
+    map = new google.maps.Map(document.getElementById("map"), {
+    //  maxZoom: 20,
+        center: munich,
+    });
+    infoWindow = new google.maps.InfoWindow()
+}
 
-/*Connecting Supermarkets locations API from EDEKA */
-const LocationApiUrl = 'https://www.edeka.de/api/marketsearch/markets?searchstring=Munchen&size=124';
-fetch(LocationApiUrl)
-.then(response => response.json())
-.then(data => {
-    /*Calling functions */
-    displayStores(data);
-    showStoresMarker(data);
-    setOnClickListener();
-    searchByPlace();
-    // searchstores();
-});     
+searchMarketsNear('München, Deutschland');
+
+/*Triggering Button when Enter is pressed */
+function triggerBtn() {
+    input = document.getElementById('inputValue');
+    input.addEventListener("keyup", function(event) {
+        if (event.keyCode === 13) {
+            event.preventDefault();
+            document.getElementById("myBtn").click();
```

Comment:
this is not the way how this should be done.
Idea when you react on Enter is not to fire click from code, but to execute the function which button is using upon click.